test/JDBC/expected/latest__verification_cleanup__13_6__sys-all_sql_modules-vu-verify.out

```diff
@@ -0,0 +1,187 @@
+USE sys_all_sql_modules_vu_prepare_db
+GO
+
+-- Test for function
+SELECT
+    definition,
+    uses_ansi_nulls,
+    uses_quoted_identifier,
+    is_schema_bound,
+    uses_database_collation,
+    is_recompiled,
+    null_on_null_input,
+    execute_as_principal_id,
+    uses_native_compilation
+FROM sys.all_sql_modules
+WHERE object_id = OBJECT_ID('sys_all_sql_modules_vu_prepare_f1')
+GO
+~~START~~
+nvarchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#bit#!#int#!#bit
+CREATE OR REPLACE FUNCTION sys_all_sql_modules_vu_prepare_db_dbo.sys_all_sql_modules_vu_prepare_f1()<newline> RETURNS integer<newline> LANGUAGE pltsql<newline>AS '{"version_num": "1", "typmod_array": ["-1"], "original_probin": ""}', $function$BEGIN<newline>    RETURN 1;<newline>END$function$<newline>#!#1#!#1#!#0#!#0#!#0#!#0#!#<NULL>#!#0
```

Comment:
This definition is as expected, right?

Reply:
For upgrade, yes. This is also related on ur comment here: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/540#discussion_r982500356 . 

Functions in general are a little bit flaky because we use the Postgres function definition and NOT one for TSQL (using pg_get_functiondef()). This was left in the original code for sql_modules **before** any SSMS-related work which causes it to use postgresql syntax (this will have to be revisited for when the TSQL definitions are implemented). 

NOTE: this is mentioned in the other comment, but when before upgrading to latest, this would say `CREATE OR REPLACE FUNCTION dbo.FUNCTION_NAME` however, when you upgrade, it replaces it with `CREATE OR REPLACE  sys_all_sql_modules_vu_prepare_db_dbo.FUNCTION_NAME`.